app/model/Permission.scala

```diff
@@ -0,0 +1,9 @@
+package model
+
+import play.api.libs.json.{Json, Writes}
+
+case class Permission(id: String, permitted: Boolean)
+
+object Permission {
```

Comment:
Why is this called permission ? 
Is this like a short version of the consents ?

Reply:
We're calling these things 'permissions' rather than 'consents' now to avoid confusing them with cookie consents etc.